### ISSUE TITLE
"1000Base-TX" network interface type missing

### ISSUE THREAD

**Issue by aaroneg** [netbox-community/netbox#15489]
### Deployment Type

Self-hosted

### NetBox Version

v3.7.4

### Python Version

3.8

### Steps to Reproduce

Edit a network interface on a device type, use the filter to type 'tx', the only type matching the filter is '100BASE-TX' which is not the same form factor. 
Documentation on the 1000-BASETX form factor can be found here https://en.wikipedia.org/wiki/Gigabit_Ethernet#1000BASE-TX
It is not enough to just say to use 1000-BASET because the two have different media requirements, they are separate.

### Expected Behavior

1000base-tx should appear as an option.

### Observed Behavior

the option does not appear

**Comment by jeremystretch** [comment]
Do you actually have 1000Base-TX equipment in use? Can you elaborate on why the 1000Base-T designation is insufficient?

**Comment by aaroneg** [comment]
Yes, and the media requirements are different as I noted when opening the
issue.

**Comment by Julio-Oliveira-Encora** [comment]
Could you please assign it to me?
